pkg/cmdimpl/add_test.go

```diff
@@ -216,7 +319,7 @@ var _ = Describe("Dry Run Add Test", func() {
 			_ = override.WithOverrides(
 				func() override.Result {
 					deps := &AddDependencies{
-						GitClient: &fakeGitClient,
+						GitClient: &failGitClient,
```

Comment:
Just to exemplify the previous comment, this could be just `GitClient: &gitfakes.FakeGit{}`

Reply:
Not for dry run